[2022-07-22 01:52:36,786] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t1 scheduled__2022-07-21T00:00:00+00:00 [queued]>
[2022-07-22 01:52:36,836] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t1 scheduled__2022-07-21T00:00:00+00:00 [queued]>
[2022-07-22 01:52:36,837] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 01:52:36,838] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-22 01:52:36,839] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 01:52:36,893] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t1> on 2022-07-21 00:00:00+00:00
[2022-07-22 01:52:36,904] {standard_task_runner.py:52} INFO - Started process 3782 to run task
[2022-07-22 01:52:36,917] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't1', 'scheduled__2022-07-21T00:00:00+00:00', '--job-id', '107', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpxao1doyi', '--error-file', '/tmp/tmpldd9grbl']
[2022-07-22 01:52:36,929] {standard_task_runner.py:80} INFO - Job 107: Subtask t1
[2022-07-22 01:52:36,972] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-22 01:52:37,153] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t1 scheduled__2022-07-21T00:00:00+00:00 [running]> on host 97e3017421e4
[2022-07-22 01:52:37,379] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2022-07-21T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-21T00:00:00+00:00
[2022-07-22 01:52:37,382] {python.py:173} INFO - Done. Returned value was: welcome
[2022-07-22 01:52:37,450] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t1, execution_date=20220721T000000, start_date=20220722T015236, end_date=20220722T015237
[2022-07-22 01:52:37,500] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 01:52:37,621] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-07-22 13:00:38,076] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t1 scheduled__2022-07-21T00:00:00+00:00 [queued]>
[2022-07-22 13:00:38,145] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t1 scheduled__2022-07-21T00:00:00+00:00 [queued]>
[2022-07-22 13:00:38,146] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 13:00:38,147] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-22 13:00:38,149] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 13:00:38,232] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t1> on 2022-07-21 00:00:00+00:00
[2022-07-22 13:00:38,249] {standard_task_runner.py:52} INFO - Started process 914 to run task
[2022-07-22 13:00:38,276] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't1', 'scheduled__2022-07-21T00:00:00+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmp0ebc6utd', '--error-file', '/tmp/tmpyab8l4wi']
[2022-07-22 13:00:38,286] {standard_task_runner.py:80} INFO - Job 100: Subtask t1
[2022-07-22 13:00:38,380] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-22 13:00:38,642] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t1 scheduled__2022-07-21T00:00:00+00:00 [running]> on host 2f63307804d5
[2022-07-22 13:00:38,959] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2022-07-21T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-21T00:00:00+00:00
[2022-07-22 13:00:38,963] {python.py:173} INFO - Done. Returned value was: welcome
[2022-07-22 13:00:39,150] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t1, execution_date=20220721T000000, start_date=20220722T130038, end_date=20220722T130039
[2022-07-22 13:00:39,222] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 13:00:39,336] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-07-25 09:18:20,748] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t1 scheduled__2022-07-21T00:00:00+00:00 [queued]>
[2022-07-25 09:18:20,789] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t1 scheduled__2022-07-21T00:00:00+00:00 [queued]>
[2022-07-25 09:18:20,790] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:18:20,791] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 09:18:20,792] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:18:20,844] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t1> on 2022-07-21 00:00:00+00:00
[2022-07-25 09:18:20,857] {standard_task_runner.py:52} INFO - Started process 858 to run task
[2022-07-25 09:18:20,863] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't1', 'scheduled__2022-07-21T00:00:00+00:00', '--job-id', '96', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmppb74l52z', '--error-file', '/tmp/tmpneohy__m']
[2022-07-25 09:18:20,870] {standard_task_runner.py:80} INFO - Job 96: Subtask t1
[2022-07-25 09:18:20,917] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 09:18:21,039] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t1 scheduled__2022-07-21T00:00:00+00:00 [running]> on host 01e6b1df1873
[2022-07-25 09:18:21,221] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2022-07-21T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-21T00:00:00+00:00
[2022-07-25 09:18:21,224] {python.py:173} INFO - Done. Returned value was: welcome
[2022-07-25 09:18:21,274] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t1, execution_date=20220721T000000, start_date=20220725T091820, end_date=20220725T091821
[2022-07-25 09:18:21,324] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 09:18:21,385] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
